src/CMakeLists.txt

```diff
@@ -60,8 +60,6 @@ if(ANDROID)
     # Core requires these options for the final *.so file. See more realm-core/CMakeLists.txt
     target_link_options(realm_dart PRIVATE LINKER:-gc-sections)
 
-    set_target_properties(realm_dart PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${PROJECT_SOURCE_DIR}/binary/android/${ANDROID_ABI}")
```

Comment:
are we sure we are outputting the files to the correct locations

Reply:
This is already being taken care of by the logic at the end of the file and it was redundant to also have it here.